lib/jormungandr/test/integration/Cardano/Faucet.hs

```diff
@@ -38,18 +38,16 @@ initFaucet =
     Faucet <$> newMVar mnemonics
 
 getBlock0H :: IO (Hash "Genesis")
-getBlock0H =
-    Hash . unsafeFromHex . T.encodeUtf8 <$> getBlock0HText
-
-getBlock0HText :: IO Text
-getBlock0HText =
-    toHex . extractId <$> BL.readFile block0
+getBlock0H = extractId <$> BL.readFile block0
   where
     block0 = "test/data/jormungandr/block0.bin"
-    extractId = getHash . runGet getBlockId
-    toHex :: BS.ByteString -> Text
-    toHex = T.decodeUtf8
-        . convertToBase @BS.ByteString @BS.ByteString Base16
+    extractId = Hash . getHash . runGet getBlockId
+
+getBlock0HText :: IO Text
+getBlock0HText = T.decodeUtf8
+    . convertToBase @BS.ByteString @BS.ByteString Base16
```

Comment:
Note from the style guide: we prefer unqualified imports for types where they are unambiguous. Also, I believe type annotation are unnecessary here because both the source and target types are known.